src/server/routes/api-routes.js

```diff
@@ -5,4 +5,24 @@ const router = express.Router();
 
 router.use('/items', items);
 
+router.use('/*', (req, res) => {
+  if (res.statusCode === 200 || res.statusCode === 304) {
+    return;
+  }
+  res.status(404).send('API Request > 404 - Page Not Found');
+});
+
+router.use((errHandler, req, res) => {
```

Comment:
I heard error handler function in Express requires 4 args(err, req, res, next).
this function will work in error situations like 400 or 500 in our item api controller?